homeassistant/components/xiaomi_miio/humidifier.py

```diff
@@ -185,6 +196,15 @@ def __init__(self, name, device, entry, unique_id, coordinator):
             self._min_humidity = 30
             self._max_humidity = 80
             self._humidity_steps = 100
+        elif self._model in MODELS_HUMIDIFIER_MJJSQ:
```

Comment:
Side note: Would be nice to update the humidifier integration to support setting these features from an entity description.